lib/logstash/filters/geoip.rb

```diff
@@ -82,139 +90,98 @@ class LogStash::Filters::GeoIP < LogStash::Filters::Base
   # of the same geoip_type share the same cache. The last declared cache size will 'win'. The reason for this is that there would be no benefit
   # to having multiple caches for different instances at different points in the pipeline, that would just increase the
   # number of cache misses and waste memory.
-  config :lru_cache_size, :validate => :number, :default => 1000
+  config :cache_size, :validate => :number, :default => 1000
```

Comment:
for backwards compatibility, can we leave `lru_cache_size` to work? and maybe call it deprecated